kyuubi-hive-jdbc-shaded/src/main/resources/META-INF/NOTICE

```diff
@@ -41,22 +41,28 @@ Curator Framework
 Copyright 2011-2017 The Apache Software Foundation
 
 Hive Common
-Copyright 2021 The Apache Software Foundation
+Copyright 2022 The Apache Software Foundation
 
 Hive JDBC
-Copyright 2021 The Apache Software Foundation
+Copyright 2022 The Apache Software Foundation
 
 Hive Metastore
-Copyright 2021 The Apache Software Foundation
+Copyright 2022 The Apache Software Foundation
+
+Hive Standalone Metastore
+Copyright 2022 The Apache Software Foundation
+
+Hive Llap Client
+Copyright 2022 The Apache Software Foundation
 
 Hive Serde
-Copyright 2021 The Apache Software Foundation
+Copyright 2022 The Apache Software Foundation
 
 Hive Service
-Copyright 2021 The Apache Software Foundation
+Copyright 2022 The Apache Software Foundation
 
 Hive Service RPC
-Copyright 2019 The Apache Software Foundation
+Copyright 2022 The Apache Software Foundation
```

Comment:
Should we also update Hive Shims 0.23 and Hive Shims Common's year?

Reply:
no, their version not updated